docs/book/features/vsphere_csi_migration.md

```diff
@@ -0,0 +1,175 @@
+Migrated vSphere volume will not get additional capabilities vSphere CSI driver supports.
+
+## Consider followings before turning on Migration
+
+- vSphere CSI Migration is released with `beta` feature-gate in Kubernetes 1.19. Refer [release note announcement](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#csi-migration---azuredisk-and-vsphere-beta).
+- Kubernetes 1.19 release has deprecated vSAN raw policy parameters for the in-tree vSphere Volume plugin and these parameters will be removed in a future release. Refer [deprecation announcement](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#deprecation)
+- Following vSphere in-tree StorageClass parameters will not be supported after enabling migration.
+  - `hostfailurestotolerate`
+  - `forceprovisioning`
+  - `cachereservation`
+  - `diskstripes`
+  - `objectspacereservation`
+  - `iopslimit`
+  - `diskformat`
+- Storage Policy should not be renamed or deleted after enabling migration. Volume migration requires storage policy to be present to successfully register volume as Container Volume in vSphere.
+- Datastore should not be renamed after enabling migration. Volume migration relies on the datastore name retrieved from the volume path available in the Persistent Volume Source.
+- For statically created vSphere in-tree Persistent Volume Claims and Persistent Volumes, make sure to add following annotations. Statically provisioned in-tree vSphere volumes can't be migrated to CSI without following these annotations.
+
+    Annotation on PV
+
+      annotations:
+        pv.kubernetes.io/provisioned-by: kubernetes.io/vsphere-volume
+
+    Annotation on PVC
+
+      annotations:
+        volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/vsphere-volume
+
+- vSphere CSI Driver does not support Provisioning `eagerzeroedthick` and `zeroedthick` volume. After the migration is enabled, when a new volume is requested using the in-tree provisioner and `diskformat` parameter, `diskformat` parameter is dropped by vSphere CSI Driver.
+- vSphere CSI Driver does not support raw vSAN policy parameters. After the migration is enabled, when a new volume is requested using in-tree provisioner and vSAN raw policy parameters specified in the StorageClass, Volume Creation will be failed by vSphere CSI Driver.
+- vSphere CSI Migration requires vSphere 7.0u1. Customers who have in-tree vSphere volumes must upgrade vSphere to 7.0u1. Customers who do not need to migrate in-tree vSphere volumes can use vSphere 67u3 and above.
+- vSphere CSI driver does not support volumes formatted with the Windows file system. Migrated in-tree vSphere volumes using the windows file system can't be used with vSphere CSI driver.
```

Comment:
Does vcp support k8s cluster on Windows nodes? If so, we need to mention k8s cluster on windows is not supported for migration.

Reply:
yes VCP supports windows nodes